lambdas/scripts/batch_update_ods_code.py

```diff
@@ -235,3 +237,9 @@ def setup_logging_for_local_script():
 if __name__ == "__main__":
     setup_logging_for_local_script()
     BatchUpdate().main()
+    print("Starting data collection process")
+    service = DataCollectionService()
+    service.collect_all_data_and_write_to_dynamodb()
+    print("Starting to create statistical report")
+    service = StatisticalReportService()
+    service.make_weekly_summary_and_output_to_bucket()
```

Comment:
Pedantic of me, but could we please name the services more specifically so that we can separate which service is doing what? Both services are using the same name currently.

Reply:
Services have now been renamed.